lxdock/provisioners/ansible.py

```diff
@@ -16,13 +16,13 @@ class AnsibleProvisioner(Provisioner):
     name = 'ansible'
 
     guest_required_packages_alpine = ['openssh', 'python', ]
-    guest_required_packages_archlinux = ['openssh', 'python2', ]
+    guest_required_packages_arch = ['openssh', 'python', ]
```

Comment:
Oh, that's true. On arch, python 2 has the `python2` executable name. That's a problem. We might as well leave thing as they are and let Ansible run under Python 3 on Arch guests. It *should* work most of the time...